### PR TITLE
Michel remove warnings

### PR DESCRIPTION
**Pull request by MichelKern**

### CODE REVIEW

File: inc/data.h [hunk 1]
```diff
@@ -47,9 +47,6 @@ namespace vt_darcy
       {
         values[p].clear ();
 
-        const double x = points[p][0];
```
Comment (mjayadharan) [inline inc/data.h:50]: x, y not used. definition used directly.

File: inc/data.h [hunk 2]
```diff
@@ -196,9 +193,6 @@ namespace vt_darcy
         if (dim == 3)
         z = p[2];
 
-        Vector<double> vec(2);
```
Comment (mjayadharan) [inline inc/data.h:199]: unknown use of vec

File: inc/projector.h [hunk 3]
```diff
@@ -20,7 +20,7 @@
 #include <deal.II/dofs/dof_tools.h>
 #include <deal.II/fe/mapping.h>
 #include <deal.II/fe/mapping_q1.h>
-#include <deal.II/lac/constraint_matrix.h>
+#include <deal.II/lac/affine_constraints.h>
```
Comment (mjayadharan) [inline inc/projector.h:23]: Could you explain this change? 
Is constraint_matrix.h a deprecated version of affine_constraints.h ?

**Review by mjayadharan — COMMENTED** [review]
All the changes looks great. It will definitely make the code and the compilation output more readable. 
If you could comment on the header file change, I could merge the pull request.

File: inc/projector.h (hunk 3 above)
Comment (MichelKern) [reply inc/projector.h:23]: Yes, that's exactly it. As of 9.1 it is deprecated, and does nothing but include the new file, so I made the change.

Comment (mjayadharan) [reply inc/projector.h:23]: Thanks. It makes sense.

**Comment by mjayadharan** [comment]
@MichelKern  did the following fixes:   
- issues related to compilation warning, including unused variables and deprecated header file.
- General formatting and indentation issues, to improve the overall readability of the code.

**Comment by mjayadharan** [comment]
Keeping the  branch in case @MichelKern  would like to make any further changes.

**Pull request merged**